prow/config.yaml

```diff
@@ -311,7 +311,6 @@ postsubmits:
     - master
   jenkins-x/jx:
   - agent: tekton
-    context: tekton
```

Comment:
Feels like we should probably remove `context: tekton` from all of the `postsubmit`s?